conbench/entities/_entity.py

```diff
@@ -167,6 +173,40 @@ def delete(self):
         Session.delete(self)
         Session.commit()
 
+    @classmethod
+    def get_or_create(cls: Type[T], props: Dict) -> T:
```

Comment:
Not sure if it's obvious what this does. This screenshot helps:
![image](https://user-images.githubusercontent.com/265630/233408921-a93b08a0-d3fd-4ce4-86dd-dc90cbc380d3.png)

The IDE Knows that the return value is of type `Context` because `Context` derives from `EntityMixin["Context"]`.